tests/inject/karma.conf.ci.js

```diff
@@ -0,0 +1,79 @@
+// @ts-check
```

Comment:
The config is now more complex and that's a bit of a problem IMO. I cannot easily spot a thing that necessary would need to move these configuration to javascript functions etc instead of just having the config file exporting the config object.

Reply:
Hm. A lot of the code is to verify the parameters coming in, and that was more of a concern when I was implementing it. I left it in as a sort of built-in regression test. 

Let me revisit and see if I can make it simpler.

Reply:
IMO this change should be trival, with mostly expecting the most of the changes to the CI and some quirky workarounds to make the CI happy. Of course is these kinds of things nice, but it's hard to review and later-on maintain and build on.

Reply:
I get it. I wasn't happy with the added complexity either, but I'd been thinking, "better here than in the Actions config". Things get weird sometimes.

Reply:
Fixed